### PR TITLE
Add Planet Learning of Open Learning Exchange

### PR DESCRIPTION
**Pull request by mappuji**
We are from [Open Learning Exchange](https://ole.org) currently building a platform for distributed learning platform for community especially where internet is constrained. We have issue for newbie or beginners.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -159,6 +159,7 @@ If you are looking to contribute then explore this list, look at [`first-timers-
 
 - [TypeScript](https://github.com/Microsoft/TypeScript/labels/Effort%3A%20Easy) _(label: Effort: Easy)_ TypeScript is a superset of JavaScript that compiles to clean JavaScript output.
 - [Visual Studio Code](https://github.com/Microsoft/vscode/labels/good%20first%20issue) _(label: good first issue)_ a new type of tool that combines the simplicity of a code editor with what developers need for their core edit-build-debug cycle.
+- [Planet Learning](https://github.com/ole-vi/planet) _(label: Newbie Friendly)_ A distributed learning platform for community especially in place where internet is a constraint.
```
Comment (vvscode) [inline README.md]: Please update link to the labeled issues

Comment (mappuji) [reply README.md]: Done @vvscode

**Comment by MunGell** [comment]
Hi @mappuji 

Thank you for your PR
As per this repository's contributing guidelines the project must already have some well-established community and link should be redirecting to a beginner-friendly issue page.

I am closing this PR for the moment, please feel free to open new PR when your project will get more mature.

**Comment by mappuji** [comment]
Thank you @MunGell . Actually we have a wiki for newbie, should I refer to that?

[Here](https://open-learning-exchange.github.io/#!pages/vi/vi-first-steps.md) is the link.

**Pull request closed without merging**